src/test/java/PlayListTests.java

```diff
@@ -80,12 +72,13 @@ public void createNameWith10Chars ()  {
         //WHEN
         playListPage.playListArea();
         playListPage.addingPlaylistbtn();
-        playListPage.enterPlayListName(tenChars);
+        playListPage.enterPlayListName("thisIsNo10");
         //THEN
-        Assert.assertEquals(playListPage.getConfirmationPopupText(), "Created playlist " + "\"" + tenChars + "." + "\"");
+        Assert.assertEquals(playListPage.getConfirmationPopupText(), "Created playlist " + "\"" + "thisIsNo10" + "." + "\"");
     }
+
     @Test(enabled = true, priority = 4)
-    public void createNameWith11Chars ()  {
+    public void createNameWith11Chars() {
```

Comment:
Test failed!

You got this problem, please correct it.

![image](https://user-images.githubusercontent.com/104761482/211770584-38614902-a271-41fd-a8a3-6f83294d54be.png)

Reply:
I changed from false to true for expected result.